hibernate-reactive-quickstart/src/main/java/org/acme/hibernate/reactive/FruitMutinyResource.java

```diff
@@ -23,67 +30,118 @@
 @Produces("application/json")
 @Consumes("application/json")
 public class FruitMutinyResource {
+    private static final Logger LOGGER = Logger.getLogger(FruitMutinyResource.class.getName());
 
     @Inject
-    Uni<Mutiny.Session> mutinySession;
+    Mutiny.Session mutinySession;
 
     @GET
-    public Uni<Fruit[]> get() {
-        return mutinySession.flatMap( session -> session.createNamedQuery( "Fruits.findAll", Fruit.class ).getResultList() )
-                .map( fruits -> fruits.toArray( new Fruit[fruits.size()] ) );
+    public Uni<List<Fruit>> get() {
```

Comment:
This is perhaps the wrong place, but given it's a quickstart to help people.. why isn't this ``Multi<Fruit>`` ? Seems like a weird API if it's only a Uni wrapper for normal blocking models instead of real reactive one.

Reply:
At the moment, hibernate reactive returns a Uni for reasons described here: https://github.com/hibernate/hibernate-reactive/issues/49

But you are right, we could return a Multi in this case. I think I will update the quickstart colling `Session#getResults` instead